pkg/util/node/node.go

```diff
@@ -187,13 +187,14 @@ func GetZoneKey(node *v1.Node) string {
 		return ""
 	}
 
-	// TODO: prefer stable labels for zone in v1.18
+	// TODO: "failure-domain.beta..." names are deprecated, but will
+	// stick around a long time due to existing on old extant objects like PVs.
+	// Maybe one day we can stop considering them (see #88493).
```

Comment:
we can still consider them as a fallback, but the goal was to give priority to the GA labels

Reply:
I don't know if that's a good idea - if they are the same value, it doesn't matter.  If they are different values - changing the priority is a breaking change for what gain?  If we are eventually going to have a break, just break it all the way